internal/render/container.go

```diff
@@ -101,22 +101,22 @@ func (c Container) Render(o interface{}, name string, r *model1.Row) error {
 		return fmt.Errorf("expected ContainerRes, but got %T", o)
 	}
 
-	cur, res := gatherMetrics(co.Container, co.MX)
+	cur, res := gatherMetrics(co.Container(), co.MX)
 	ready, state, restarts := "false", MissingValue, "0"
-	if co.Status != nil {
-		ready, state, restarts = boolToStr(co.Status.Ready), ToContainerState(co.Status.State), strconv.Itoa(int(co.Status.RestartCount))
+	if co.Status() != nil {
+		ready, state, restarts = boolToStr(co.Status().Ready), ToContainerState(co.Status().State), strconv.Itoa(int(co.Status().RestartCount))
 	}
 
-	r.ID = co.Container.Name
+	r.ID = co.IndexLabel()
 	r.Fields = model1.Fields{
-		co.Container.Name,
+		co.IndexLabel(),
+		co.Container().Name,
```

Comment:
We are recomputing the container 5 times vs just calling the member. I don't think we should lazy eval this call.